semantic/src/themes/universe/modules/modal.variables

```diff
@@ -0,0 +1,28 @@
+/* Header */
+@headerFontSize: 20px;
+@headerBorder: none;
+@headerFontWeight: 500;
+@headerVerticalPadding: 24px;
+@headerHorizontalPadding: 24px;
+
+/* Content */
+@contentLineHeight: 20px;
+@contentPadding: 0 24px 24px;
+
+/* Modal Actions */
+@actionBackground: #FFFFFF;
+@actionBorder: none;
+@actionPadding: 0 24px 24px;
+
+/* Close Icon */
+@closeTop: 24px;
+@closeRight: 24px;
+@closeColor: #a9b0b7;
```

Comment:
no big deal, but do we have global vars for these colours?

Reply:
updated to `@inkLighter` 👍